src/test/java/apprenticeship/BoardTest.java

```diff
@@ -16,44 +14,72 @@ public void initBoard() {
         tictactoe = new Board();
     }
 
-    @Test
-    public void aBoardInitializeWithNineCells() throws Exception {
-        int emptyCells = tictactoe.countCells();
-        assertEquals(emptyCells, 9);
-    }
 
     @Test
     public void canInsertSymbolOnBoard() throws Exception {
-        tictactoe.insertSymbol('x', 0);
-        char atZero = tictactoe.getSymbol(0);
-        assertEquals(atZero, 'x');
+        tictactoe.insertSymbol("X", 0);
+        String atZero = tictactoe.getSymbol(0);
```

Comment:
because I want to be able to print the cell number on the grid. I am thinking later on when I have a player, I will need to have those cell number there for the human to identify the cell they want to pick.